package-lock.json

```diff
@@ -2509,6 +2509,28 @@
         "node": ">=6.0.0"
       }
     },
+    "node_modules/@jridgewell/source-map": {
```

Comment:
Why there's a change in package-lock.json file ? 
These files shouldn't be added for PR.